client/client.go

```diff
@@ -342,6 +345,29 @@ func main() {
 		log.Infof("=============================================================================")
 	}
 
+	// Reconnect to the server with the provided server_trust_cert.
```

Comment:
Why do we do this? Did we not have the cert before? :O

Reply:
Yep, the first connection is untrusted. We can't make it trusted initially as the server's cert is self-signed. It's kinda documented in step 6 of https://github.com/openconfig/bootz#api-flow.